containerfiles/README.md

```diff
@@ -63,10 +65,21 @@ podman run --rm -ti \
     -w /home/prow \
     -e HOME=/home/prow \
     -v ~/.ssh/id_ed25519:/home/prow/.ssh/id_ed25519 \
-    -v .:/home/prow/ci-framework \
-    -v ~/inventory.yml:/home/prow/ci-framework/inventory.yml \
+    -v $(pwd):/home/prow/ci-framework \
+    -v ~/custom_files:/home/prow/custom_files \
     --security-opt label=disable \
     localhost/cifmw:latest bash
 ```
 
+##### On systems utilizing Podman Remote Machine (MacOS, Windows, ...):
```

Comment:
shouldn't we point to the fact we need to init the podman machine with a specific option to expose the mounts?

Reply:
You're right, thanks for letting me know it is missing.